airqo-platform/AirQo-frontend
# Adding the data export sidebar item

- What does this PR do?
This PR introduces the data export sidebar option inside the platform

- Which JIRA board item is attached to this PR?
[This one
](https://airqoteam.atlassian.net/browse/PLAT-88?atlOrigin=eyJpIjoiY2JkOWY4ZmRlYWE1NDc1Nzg5NjQwZWFkZjgwZWYyMTgiLCJwIjoiaiJ9)
- How can I test it out?
cd AirQo-frontend/netmanager
npm install

> for MAC users

```
npm run dev-mac
npm run stage-mac
npm run prod-mac
```

> for Windows users

```
npm run dev-pc
npm run stage-pc
npm run prod-pc
```

> Alternatively:
> You can test out using Docker:

```
npm run build
docker build -t us.gcr.io/airqo-frontend/airqo-stage-netmanager-frontend .
docker run -d -p 8888:80 us.gcr.io/airqo-frontend/airqo-stage-netmanager-frontend
```

- Under what environment should I test it out?
node v12.16.1
